<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>std::vector demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="list_widget"/>
      </item>
      <item>
       <widget class="QFrame" name="verticalFrame">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="verticalFrame">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="txt_elem_content"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Методы</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="0">
               <widget class="QPushButton" name="pb_clear">
                <property name="text">
                 <string>clear</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="pb_pop_back">
                <property name="text">
                 <string>pop_back</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pb_push_back">
                <property name="text">
                 <string>push_back</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Заготовки</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pb_weekdays">
                <property name="text">
                 <string>= дни недели</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_months">
                <property name="text">
                 <string>= месяцы</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignBottom">
          <widget class="QFrame" name="horizontalFrame_3">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>size()</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_size"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
